projects/angular-d3-cloud/src/lib/angular-d3-cloud.component.ts

```diff
@@ -89,27 +92,55 @@ export class AngularD3CloudComponent implements OnChanges, OnInit {
             }
           })
           .attr('text-anchor', 'middle')
-          .attr('transform', d => `translate(${[d.x, d.y]})rotate(${d.rotate})`)
-          .text((d: any) => d.text)
-
-        if (this.isMouseClickUsed) {
-          texts.on('click', (event: MouseEvent, word: cloud.Word) => {
-            this.wordClick.emit({ event, word })
-          })
-        }
-        if (this.isMouseOverUsed) {
-          texts.on('mouseover', (event: MouseEvent, word: cloud.Word) => {
-            this.wordMouseOver.emit({ event, word })
-          })
-        }
-
-        if (this.isMouseOutUsed) {
-          texts.on('mouseout', (event: MouseEvent, word: cloud.Word) => {
-            this.wordMouseOut.emit({ event, word })
-          })
-        }
-      })
-
+          .text((d: any) => d.text);
+
+          if (!this.animations) {
+            texts
+            .attr('transform', d => `translate(${[d.x, d.y]})rotate(${d.rotate})`)
+            .style('font-size', d => `${d.size}px`)
+            .style("fill-opacity", 1)
+
+          } else {
+            // Initial status
+            texts
+            .style('font-size', 1)
+            .style("fill-opacity", 1e-6);
+
+            //Entering and existing words
+            texts
+            .transition()
+                .duration(600)
+                .attr('transform', d => `translate(${[d.x, d.y]})rotate(${d.rotate})`)
+                .style('font-size', d => `${d.size}px`)
+                .style("fill-opacity", 1);
+
+            //Exiting words
+            texts
```

Comment:
When any update happen with this component, HTML elements will be cleared immediately and exit animation won't ever run